doc/api.rst

```diff
@@ -6,7 +6,8 @@
 API Documentation
 =================
 
-MNE BIDS (:py:mod:`mne_bids`):
+MNE BIDS (:py:mod:`mne_bids`)
+=============================
```

Comment:
can you minimize the number of changes not related to PR? It makes the diff harder to read ...